### PR TITLE
gnrc_networking example: add capability to send UDP packets repeatedly

### PR DESCRIPTION
**Pull request by authmillenon**
This allows for sending UDP packets repeatedly with a configured delay.

### CODE REVIEW

**Comment by cgundogan** [comment]
rebase, please

**Comment by PeterKietzmann** [comment]
Still needs rebase. Will ACK then

**Comment by authmillenon** [comment]
Rebased and ported from vtimer to xtimer

File: examples/gnrc_networking/udp.c [hunk 1]
```diff
@@ -53,40 +53,38 @@ static void send(char *addr_str, char *port_str, char *data)
...
+        }
+        /* send packet */
+        if (!gnrc_netapi_dispatch_send(GNRC_NETTYPE_UDP, GNRC_NETREG_DEMUX_CTX_ALL, ip)) {
+            puts("Error: unable to locate UDP thread");
+            gnrc_pktbuf_release(ip);
+            return;
+        }
+        printf("Success: send %u byte to %s:%u\n", (unsigned)payload->size,
+               addr_str, tmp);
+        xtimer_usleep(delay);
```
Comment (PeterKietzmann) [inline examples/gnrc_networking/udp.c]: Seems this does not work on samr21-xpro

**Comment by PeterKietzmann** [comment]
Please adapt the commit message to `gnrc` and maybe the title of the PR

**Comment by authmillenon** [comment]
Done

**Comment by PeterKietzmann** [comment]
There are troubles in the msp430 build-test group. @OlegHahm any ideas why?

**Comment by authmillenon** [comment]
Because those are 16-bit platforms and `1000000 > (1 << 16)`. Anyway, fixed and squashed immediately.

**Comment by PeterKietzmann** [comment]
ah, sure...

**Comment by PeterKietzmann** [comment]
ACK and go

**Pull request merged**